src/calendar/day/index.tsx

```diff
@@ -4,28 +4,39 @@ import XDate from 'xdate';
 import memoize from 'memoize-one';
 
 import React, {Component} from 'react';
-
+// @ts-expect-error
 import {shouldUpdate} from '../../component-updater';
+// @ts-expect-error
 import dateutils from '../../dateutils';
+// @ts-expect-error
 import {xdateToData} from '../../interface';
+// @ts-expect-error
 import {SELECT_DATE_SLOT} from '../../testIDs';
-import BasicDay from './basic';
+import BasicDay, {BasicDayProps} from './basic';
 import PeriodDay from './period';
+import {MarkingProps} from './marking';
+
+const basicDayPropsTypes = _.omit(BasicDay.propTypes, 'date');
```

Comment:
Why the change of name if you're not using `BasicDayProps` to get the 'date'?

Reply:
I found it confusing to have both `basicDayProps` and `BasicDayProps`

Reply:
I understand but why are you still using `BasicDay.propTypes`?

Reply:
This is for the Day propTypes - as I mentioned, we decided to keep all the propTypes for now...

Reply:
I understand why you're keeping the propTypes, I'm not sure why you're using it here...

Reply:
The Day propTypes is using the BasicDay.propTypes (as before) and the DayProps is using the BasicDayProps.